Jenkinsfile.integration

```diff
@@ -24,6 +24,7 @@ pipeline {
         /* first value in the list is the default */
         choice(choices: ['airship', 'osh'], description: 'Which deployment mechanism?', name: 'deployment')
         choice(choices: ['caasp3', 'caasp4'], description: 'Which caasp version to use?', name: 'caasp_version')
+        choice(choices: ['ses', 'ses_rook'], description: 'Which ses version to use?', name: 'ses_version')
```

Comment:
What about `ses_aio` (or `ses_all_in_one`) instead of just `ses`?

Reply:
good to me

Reply:
fixed